mongodb-sync-v4-driver/src/main/java/com/github/cloudyrock/mongock/driver/mongodb/sync/v4/driver/MongoSync4DriverBase.java

```diff
@@ -10,11 +11,13 @@
 import com.mongodb.client.TransactionBody;
 
 import java.util.Optional;
+import java.util.Set;
 
 @NotThreadSafe
-public abstract class MongoSync4DriverBase<CHANGE_ENTRY extends ChangeEntry> extends MongoSync4DriverGeneric<CHANGE_ENTRY> {
+public  class MongoSync4DriverBase<CHANGE_ENTRY extends ChangeEntry> extends MongoSync4DriverGeneric<CHANGE_ENTRY> {
```

Comment:
In driver 4 you removed "abstract class", but in driver 3 don't. why?

Reply:
I did remove the abstract temporally for the compiler to tell me which method is missing, because I wanted to implemented there. Then I restore the abstract, but I forgot about it.

Good spot it!